### ISSUE TITLE
CS1061: 'InputActions.DebugActions' does not contain a definition for 'Assert'

### ISSUE THREAD

**Issue by gigcastro** [edvart-ros/unity_asv_sim#23]
Hi, congrats on the simulator, looks awesome!

I followed the wiki and installed the proper unity version 2023.2.20f1 and HDRP.

Project and assets load, but cannot execute it as it throws:
Assets/Input System/InputActions.cs(303,15): error CS1061: 'InputActions.DebugActions' does not contain a definition for 'Assert' and no accessible extension method 'Assert' accepting a first argument of type 'InputActions.DebugActions' could be found (are you missing a using directive or an assembly reference?)

Looking around seems that there may be naming conflicts with Input System 1.8.2:
[(Character Controller package install) 'Debug' does not contain a definition for 'Assert'](https://forum.unity.com/threads/inputsystem-1-8-0-causes-compile-error-does-not-contain-a-definition-for-assert.1538644/)

I'm don't really have much experience in Unity, I'm using Ubuntu 22.04.
Could someone help me workaround this?

Thanks.

**Comment by edvart-ros** [comment]
Yeah, this is a known bug that occured after some version change that went unnoticed (my bad). Seems like a naming conflict, yeah. You *could* simply comment out the "assert" lines, do the renaming thing mentioned in the forum or remove the input system altogether if you dont need it.
